File: BCE/202505/24/boj_s5_1789_수들의합.cs
using System;

public class boj_s5_1789_수들의합 // 백준 클래스 이름 Program으로 작성해야 함
{
	public static void Main(string[] args)
	{
		ulong S = ulong.Parse(Console.ReadLine());
		ulong sum = 0;
		ulong N = 0;
		while(sum <= S)
		{
			sum += ++N;
		}
		Console.WriteLine(N - 1);
	}
}

File: BCE/202505/24/boj_s5_1789_수들의합.java
package a20230524;

import java.io.*;

public class boj_s5_1789_수들의합 {

	static long S;
	
	public static void main(String[] args) throws Exception {

		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		S = Long.parseLong(br.readLine());
		long sum = 0;
		long num = 0;
		while(sum <= S) {
			sum += ++num;
		}
		br.close();
		System.out.println(num-1);
	}

}
